docs/source/daal/algorithms/outlier_detection/univariate.rst

```diff
@@ -81,9 +84,12 @@ For more details, see :ref:`algorithms`.
     If you do not provide at least one of the ``location``, ``scatter``, ``threshold`` inputs,
     the library will initialize all of them with the following default values:
 
-    .. list-table::
+    .. tabularcolumns::  |\Y{0.3}|\Y{0.7}|
```

Comment:
As discussed, I recommend NOT embedding `list-table` directive inside any other directive. 
Instead, keep syntax simple, like so, by changing  the `note` to a paragraph: 

```

If you do not provide at least one of the ``location``,
``scatter``, ``threshold`` inputs,
the library will initialize all of them with the following default
values:


.. tabularcolumns::  |\Y{0.3}|\Y{0.7}|

.. list-table:: Default Values for Algorithm Input of Univariate Outlier Detection (Batch Processing)
    :widths: 10 20
    :class: longtable

    * - ``location``
      - A set of :math:`0.0`

    * - ``scatter``
      - A set of :math:`1.0`

    * - ``threshold``
```

In general, try not to embed directives within directives. If done, it increases the probability of error w/ LaTeX builder.